Fix the following build error:

  g++: error: unrecognized command-line option '--param lto-partitions=1'; did you mean '--param lto-partitions='?

(this needs to be passed as two arguments, rather than a single arg
containing a space)

--- a/security/sandbox/linux/moz.build
+++ b/security/sandbox/linux/moz.build
@@ -117,7 +117,7 @@ if CONFIG["CC_TYPE"] in ("clang", "gcc")
 # forcing there to be only one partition.
 for f in CONFIG["OS_CXXFLAGS"]:
     if f.startswith("-flto") and CONFIG["CC_TYPE"] != "clang":
-        LDFLAGS += ["--param lto-partitions=1"]
+        LDFLAGS += ["--param", "lto-partitions=1"]
 
 DEFINES["NS_NO_XPCOM"] = True
 DisableStlWrapping()
